htdocs/compta/prelevement/class/bonprelevement.class.php

```diff
@@ -895,20 +895,17 @@ public function create($banque = 0, $agence = 0, $mode = 'real', $format = 'ALL'
 			$sql .= ", pfd.code_banque, pfd.code_guichet, pfd.number, pfd.cle_rib";
 			$sql .= ", pfd.amount";
 			$sql .= ", s.nom as name";
+			$sql .= ", f.ref, sr.bic, sr.iban_prefix, sr.frstrecur";
 			if ($type != 'bank-transfer') {
 				$sql .= " FROM ".MAIN_DB_PREFIX."facture as f";
+				$sql .= " LEFT JOIN " . MAIN_DB_PREFIX . "prelevement_facture_demande as pfd ON f.rowid = pfd.fk_facture";
```

Comment:
Starting with v18, the table prelevement_facture_demande has been renamed into prelevement_demande

Reply:
Fixed

Reply:
Should I change `pfd` to `pd` or `pld` ???

Reply:
"pfd" into "pd" would be perfect but not so important

Reply:
Done